Source/Shapes/Line.cpp

```diff
@@ -0,0 +1,18 @@
+#include "Ashkal/Shapes/Line.hpp"
+#include <climits>
+
+using namespace Ashkal;
+
+Line::Line() {
+  Point point;
```

Comment:
Use a delegating constructor:

https://en.cppreference.com/w/cpp/language/initializer_list

Initialize the Point passed into the delegated constructor using aggregate initialization:

https://en.cppreference.com/w/cpp/language/aggregate_initialization